pkg/provider/akamai/akamai.go

```diff
@@ -168,7 +168,7 @@ func (oc *Client) Authenticate(loginDetails *creds.LoginDetails) (string, error)
 	authStatus := gjson.Get(resp, "status").String()
 	if authStatus != "200" {
 		authFailReason := gjson.Get(resp, "msg").String()
-		fmt.Printf("Login Failed %s\n", authFailReason)
+		log.Printf("Login Failed %s\n", authFailReason)
```

Comment:
Actually that one is a duplicate from the line below. you can remove it altogether or delete the next line.